pages/dkp/kommander/2.1/security/howto-dex-saml-connector/index.md

```diff
@@ -0,0 +1,115 @@
+
+You will provide the issuer URL and the Assertion Consumer Service (ACS) or callback URL to your IdP.
+
+The issuer URL points to the authentication endpoint at the service provider (Dex) that issues a request towards the IdP via the user agent.
+
+The issuer URL follows this schema:
+
+```text
+https://<your-cluster-host>/dex
+```
+
+The ACS URL points to the service provider (Dex) endpoint that will receive SAML assertions issued by the IdP.
+
+The ACS or callback URL will look like this:
+
+```text
+https://<your-cluster-host>/dex/callback
+```
+
+Depending on the IdP, you may be asked to provide the configuration in some form of an XML snippet - see an example below;
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<EntityDescriptor xmlns="urn:oasis:names:tc:SAML:2.0:metadata" entityID="https://<your-cluster-host>/dex">
+   <SPSSODescriptor AuthnRequestsSigned="false" WantAssertionsSigned="true" protocolSupportEnumeration="urn:oasis:names:tc:SAML:2.0:protocol">
+      <NameIDFormat>urn:oasis:names:tc:SAML:2.0:nameid-format:persistent</NameIDFormat>
+      <AssertionConsumerService index="0" isDefault="true" Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="https://<your-cluster-host>/dex/callback" />
+   </SPSSODescriptor>
+</EntityDescriptor>
+```
+
+Step 3: Modify the `dex` configuration:
+
+For this step, you will need to get the following from your IdP:
+
+- single sign-on URL or SAML URL -> `ssoURL`
+- base64 encoded, PEM encoded CA certificate -> `caData`
+- username attribute name in SAML response -> `usernameAttr`
+- email attribute name in SAML response -> `emailAttr`
+
+From step 2 you need:
+
+- issuer URL -> `entityIssuer`
+- callback URL -> `redirectURI`
+
+Make sure you base64 encode the contents of the PEM file. As an example, the prefix of the contents will result into this exact base64 prefix:
+
+```text
+LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tC[...]
+```
+
+You can add the configuration as the values field in the `dex` application. An example `dex` configuration provided to the Kommander CLI’s install command would look similar to this:
+
+```yaml
+apiVersion: config.kommander.mesosphere.io/v1alpha1
+kind: Installation
+apps:
+  dex:
+    values: |
+      config:
+        connectors:
+        - type: saml
+          id: saml
+          name: SAML
+          config:
+            ssoURL: <url for POST request>
+            caData: < base64 PEM encoded CA for the IdP server >
+            redirectURI: https://<your-cluster-host>/dex/callback
+            entityIssuer: https://<your-cluster-host>/dex
+            usernameAttr: < user attribute in saml response >
+            emailAttr: < email attribute in saml response >
+[...]
+```
+
+Step 4: Modify the `traefik-foward-auth-mgmt` configuration and add a whitelist:
+
+This step is required to give access to a user to ops-portal. For each user, you must [give access to Kubernetes resources](../rbac) and add an entry in the `whitelist` below.
```

Comment:
```suggestion
This step is required to give access to a user to the Kommander dashboard. For each user, you must [give access to Kubernetes resources](../rbac) and add an entry in the `whitelist` below.
```

Should we be calling the UI "Kommander dashboard" or something like "web interface" ? (I've seen both)

Reply:
I can ask in the dkp-ux channel

Reply:
@gvasquezvargas should I just use `Kommander UI` here?

Reply:
I guess it depends. Is the UI's wording already been changed to DKP? If yes, I would rather change it to DKP UI already, but if it still called Kommander, I would leave it that way. @cmastr what do you think?

Reply:
Changing this to `DKP UI` based on this Slack thread, but if this is wrong for some reason, let me know: https://mesosphere.slack.com/archives/C052HL47U/p1647015731820569